production/helm/loki/templates/read/statefulset-read.yaml

```diff
@@ -100,6 +100,8 @@ spec:
           volumeMounts:
             - name: config
               mountPath: /etc/loki/config
+            - name: runtime-config
```

Comment:
Same comments as for the `statefulset-write.yaml` file

Reply:
done :)